### PR TITLE
Reuse OLM installer.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Reuse OLM installer.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16862

### Depends on:
https://github.com/che-incubator/chectl/pull/724

### How to test:

```shell
# Generate files update.
$ ./sync-chectl-to-crwctl.sh /home/user/projects/chectl/ /home/user/codeready-workspaces-chectl/ 2.2.0 2.2.0

$ cd bin

# Install CodeReady with specific version:
$ ./run server:start -a olm -n moon45 --starting-csv crwoperator.v2.1.0  --self-signed-cert

# Update CodeReady:
$ ./run server:update -a olm -n moon45

# Uninstall:
$ ./run server:delete -n moon45
```

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

**Comment by AndrienkoAleksandr** [comment]
https://github.com/che-incubator/chectl/pull/724 merged.

File: sync-chectl-to-crwctl.sh [hunk 1]
```diff
@@ -89,7 +98,7 @@ platformString="    platform: string({\n\
 installerString="    installer: string({\n\
       char: 'a',\n\
       description: 'Installer type',\n\
-      options: ['operator'],\n\
+      options: ['operator', 'olm'],\n\
       default: 'operator'\n\
```
Comment (nickboldt) [inline sync-chectl-to-crwctl.sh:102]: should olm be the new default?

Comment (tolusha) [reply sync-chectl-to-crwctl.sh:102]: for stable version of `chectl` olm is default installler for openshift 4.x f

File: sync-chectl-to-crwctl.sh [hunk 2]
```diff
@@ -55,14 +55,23 @@ pushd "${SOURCEDIR}" >/dev/null
...
 			\
+			-e "s|(const CHE_CLUSTER_CR_NAME =).+|\1 'codeready-workspaces'|g" \
+			\
+			-e "s|(const DEFAULT_CHE_OLM_PACKAGE_NAME =).+|\1 'codeready-workspaces'|g" \
+			-e "s|(const OLM_STABLE_CHANNEL_NAME =).+|\1 'latest'|g" \
+			-e "s|(const CUSTOM_CATALOG_SOURCE_NAME =).+|\1 'codeready-custom-catalog-source'|g" \
+			-e "s|(const SUBSCRIPTION_NAME =).+|\1 'codeready-subscription'|g" \
+			-e "s|(const OPERATOR_GROUP_NAME =).+|\1 'codeready-operator-group'|g" \
+			-e "s|(const OPENSHIFT_OLM_CATALOG =).+|\1 'redhat-operators'|g" \
+			-e "s|(CVS_PREFIX =).+|\1 'crwoperator'|g" \
```
Comment (nickboldt) [inline sync-chectl-to-crwctl.sh:73]: should this be CSV?

**Pull request merged**